week1/list1_done.py

```diff
@@ -0,0 +1,111 @@
+from operator import itemgetter
+# Given a list of strings, return the count of the number of
+# strings where the string length is 2 or more and the first
+# and last chars of the string are the same.
+# Note: python does not have a ++ operator, but += works.
+
+
+def match_ends_with_lambda(words):
+    length_list = list(filter(lambda x: len(x) >= 2, words))
+    list_to_return = list(filter(lambda x: x[0] == x[-1], length_list))
+    return len(list_to_return)
+
+
+def match_ends_without_lambda(words):
```

Comment:
Please implement this using [list comprehension](https://docs.python.org/2/tutorial/datastructures.html#list-comprehensions).